_posts/2022-11-07-changelog-182.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #182"
+date: 2022-11-07 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8755] Support `let_chains` (by [@Kobzol])
+
+* [#9105] Allow to run a specific [documentation test](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html) [all the ways](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-testing.html) you can run a usual test (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-182/run_doctest" w="700px" %}
+
+* [#9349] Decorate MSVC type names when rendering values in the debugger on Windows. Now the Rust type names containing MSVC-specific wrappers are rendered properly, e.g. `tuple$<Foo, ref$<Bar>>` is replaced by `(Foo, &Bar)`.
+  This feature can be disabled via the `Decorate MSVC type names` checkbox in the `File | Settings | Build, Execution, Deployment | Debugger | Data Views | Rust` settings
+
+  <img src="/assets/posts/changelog-182/msvc_decorators.png" width="700px"/>
```

Comment:
Maybe just `After` instead of `With decorator`?
I understand that you can get the same behavior as in `Before` even after the corresponding changes, but it's quite inconsistent